book/plugins/general-abilities/multiple-types-one-repo/multiple-types-one-repo.md

```diff
@@ -0,0 +1,15 @@
+# One Code Base for Multiple Plugins
+
+A developer can open one repository for handling different plugin types.
+
+For example, you can build a push plugin and later add support for an analytic plugin in the same repository.
```

Comment:
an analytic -> analytics
Same comment about repository.
The main point is they use the same codebase , library / framework / dependancies etc

Reply:
10x done